README.md

```diff
@@ -7,7 +7,7 @@ Facilitator init command will create facilitator config for an auxiliary chain.
 A simple run would be the following:
 
 ```
-./facilitator init --mosaic-config <mosaic-config> --chain-id <chain-id> --origin-password <origin-password> --auxiliary-password <auxiliary-password> --origin-rpc <origin-rpc> --auxiliary-rpc <auxiliary-rpc> --db-path <db-path> --force
+./facilitator init --mosaic-config <mosaic-config> --chain-id <chain-id> --origin-password <origin-password> --auxiliary-password <auxiliary-password> --origin-rpc <origin-rpc> --auxiliary-rpc <auxiliary-rpc> --origin-graph-ws <origin-graph-ws> --origin-graph-rpc <origin-graph-rpc> --auxiliary-graph-ws <auxiliary-graph-ws> --auxiliary-graph-rpc <auxiliary-graph-rpc> --db-path <db-path> --force
```

Comment:
In code `--chain-id` is changed to `--aux-chain-id`, Read me still says `--chain-id`.